Code/GraphMol/Wrap/Mol.cpp

```diff
@@ -248,6 +248,7 @@ struct mol_wrapper {
         .value("PrivateProps", RDKit::PicklerOps::PrivateProps)
         .value("ComputedProps", RDKit::PicklerOps::ComputedProps)
         .value("AllProps", RDKit::PicklerOps::AllProps)
+        .value("ConfDouble", RDKit::PicklerOps::ConfDouble)
```

Comment:
Now that we know it's possible, I think these should have docstrings (copied from the C++)

Reply:
boost::python::enum_ apparently can't handle docstrings